tests/python/gpu/test_operator_gpu.py

```diff
@@ -45,6 +45,7 @@
 from test_sparse_ndarray import test_sparse_nd_setitem, test_sparse_nd_binary_scalar_op
 from test_sparse_operator import *
 from test_ndarray import *
+from test_summary import *
```

Comment:
Does this really have to run on a GPU? In general, this seems like an integration test to me - I think it would be better to have it in the Integration stage of our Jenkins pipeline

Reply:
Yes, because we need to verify the logging APIs can take care of `NDArray`s allocated on GPUs as correctly as on CPUs. Logging is just the final step for those APIs. There are also data processing algorithms implemented for different APIs.

Reply:
Great, thanks